ArchiSteamFarm/Bot.cs

```diff
@@ -150,6 +150,10 @@ public sealed class Bot : IDisposable {
 		private string TwoFactorCode;
 		private byte TwoFactorCodeFailures;
 
+		internal bool HasWallet { get; private set; }
```

Comment:
Do we really need this? I'd just display 0 in currency instead.

Reply:
You can check what `ECurrencyCode` of `0` is, if it's unknown, you can use that info for no wallet, if it's not unknown, you can make `WalletCurrency` `ECurrencyCode?` instead and check for `null`.

Reply:
If bot has no wallet Currency = "Invalid", display as "0 invalid"

Reply:
@MisakaDev Very good! Then simply remove the `bool` you've added and use this for no wallet, the message can stay.